src/utils.py

```diff
@@ -0,0 +1,33 @@
+"""Utils"""
+from copy import copy
+
+import pandas as pd
+
+
+def add_prefixes_to_plain_id(x: str) -> str:
+    """From plain IDs from originanl source, add prefixes.
+
+    Terms:
+        CN: stands for "CUI Novel". These are created for any MedGen records without UMLS CUI.
+        C: stands for "CUI". These are sourced from UMLS.
+        CUI: stands for "Concept Unique Identifier"
+        UID (Unique IDentifier): These are cases where the id is all digits; does not start with a leading alpha char.
+    """
+    return f'MEDGENCUI:{x}' if x.startswith('CN') \
```

Comment:
## Refactoring
I was considering doing this PR along with #15, but good to separate, especially since I needed to refactor here.

Basically I previously had a function that would read in `MedGenIDMappings.txt` and convert it to what we wanted.

While I was doing that, I wanted to move this logic somewhere else, like to a simple key value JSON or a 2 column TSV:
```py
    return f'MEDGENCUI:{x}' if x.startswith('CN') \
        else f'UMLS:{x}' if x.startswith('C') \
        else f'MEDGEN:{x}'
```

But I found that representing this in such a format was not clear. Because these steps have to each be run sequentially, and the same operation is not performed in each step.

Reply:
FYI, I checked to see all of the sources that actually have a mapping to a MEDGENCUI / CN (CUI Novel), and these are the only ones:
- GARD
- OMIM allelic variant
- OMIM Phenotypic Series

Reply:
Also, after working on #16, it looks like the robot template logic doesn't use `add_prefixes_to_plain_id()` anymore. Right now, the prefixes are assigned correctly a different way. But I feel intuitively like we should be using this mapping/prefix assignment somewhere in the codebase.

Reply:
Regarding contingent MEDGENCUI preservation

Options discussed:
a. Test that fails if any MEDGENCUI appears
b. Let them slip in with the rest of the UMLS / MEDGEN.
c. Filter them out. Write them to a separate file. Or log to a file.
d. A GH action that reviews the release and detects if MEDGENCUI and throw warning.